app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/OrganisationDetailsTypeController.scala

```diff
@@ -110,17 +116,19 @@ class OrganisationDetailsTypeController @Inject() (
       )
     )
 
-  private def getUkCompanyRedirectUr()(implicit
-    request: JourneyRequest[AnyContent]
-  ): Future[String] =
-    ukCompanyGrsConnector.createJourney(
-      UkCompanyGrsCreateRequest(appConfig.grsCallbackUrl,
-                                Some(request2Messages(request)("service.name")),
-                                appConfig.serviceIdentifier,
-                                appConfig.externalSignOutLink
-      )
+  private def incorpEntityGrsCreateRequest(implicit request: Request[_]) =
+    IncorpEntityGrsCreateRequest(appConfig.grsCallbackUrl,
+                                 Some(request2Messages(request)("service.name")),
+                                 appConfig.serviceIdentifier,
+                                 appConfig.externalSignOutLink
     )
 
+  private def getUkCompanyRedirectUr()(implicit request: Request[_]): Future[String] =
+    ukCompanyGrsConnector.createJourney(incorpEntityGrsCreateRequest)
+
+  private def getRegisteredSocietyRedirectUr()(implicit request: Request[_]): Future[String] =
```

Comment:
Typo

Reply:
You must have excellent eyesight and a decent monitor.  Took me a while to spot even with you calling it out.

Reply:
Fixed